app/questionnaire/placeholder_transforms.py

```diff
@@ -19,6 +19,7 @@ def __init__(self, language):
 
     locale = DEFAULT_LOCALE
     input_date_format = '%Y-%m-%d'
+    input_date_format_md_only = '%Y-%m'
```

Comment:
Can we use a variable name here that isn't abbreviated?

Reply:
Oh you python types and your love of widescreen code. If you prefer it, I'll do that.

Reply:
Generally the style guide in runner is to avoid abbreviations if at all possible.